docs/source/developers/cpp/building.rst

```diff
@@ -424,6 +436,8 @@ this can be accomplished with the ``Threads`` built-in package:
    find_package(Threads REQUIRED)
    target_link_libraries(my_target PRIVATE Threads::Threads)
 
+.. _label-extra-debugging:
```

Comment:
```suggestion
.. _cpp-extra-debugging:
```

(I propose to leave out "label", since this is indeed a label, but we generally don't include that in the label's name, but adding cpp might be more explicit when referencing this elsewhere)